tools/azure-devtools/src/azure_devtools/perfstress_tests/perf_stress_test.py

```diff
@@ -6,6 +6,11 @@
 import os
 import aiohttp
 
+try:
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin
```

Comment:
This try/except around the import is probably unnecessary - as the perfstress framework can only be run under Python 3 anyway, as it's natively async.